pkg/adaptor/elasticsearch/clients/registry.go

```diff
@@ -0,0 +1,36 @@
+package clients
```

Comment:
I can see us having/wanting to use this sort of registry for other db types outside of ES. Let's make an issue for making this generic and have the other adaptors use the registry as well. I guess we would need to be able to get the version information through an API call like we do for ES. Should be possible.